ChongaksGame.js

```diff
@@ -0,0 +1,217 @@
+        console.log("")
+        console.log("attack potion")
+        console.log("health potion")
+        console.log("defense potion")
+        potion = readLineSync.question("You can only have one we are low on supplies!")
+        potion = potion.trim().toLowerCase()
+        
+    }switch (potion) {
+        case "attack potion":
+        console.log("")
+        console.log("This attack potion will give you a boost to your power")
+        break;
+        case "health potion":
+        console.log("")
+        console.log("This health potion will give you a boost to your health")
+        break;
+        case "defense potion":
+        console.log("")
+        console.log("This defense potion will allow you to take more hits")
+    }
+    
+}
+
+const offHandChoice = (offHand) => {
+    if(readLineSync.keyInYN("Oh you're missing an offhand would you like to grab an offhand? \n")) {
+        let offHand = ["Shield ", "Sword", "Axe", "Dagger"];
+        indexoffHand = readLineSync.keyInSelect(offHand, "So what will you bring for your offhand?")
+        console.log("")
+        console.log("This " + offHand[indexoffHand] + " will provide support for you in battle")
+    } else {
+        console.log("Alright i respect your decision lets move on.")
+    }
+    
+}
+
+const allyChoice = (ally) => {
+    if(readLineSync.keyInYN("There arent many of us available for this fight so we're moving in groups of two, would you like a partner to watch your back? \n")){
+        console.log("Alright please let me know who will join you on your fight with Chongaks")
+        let ally = ["Margo the wizard" , "Remornia the archer"]
+        indexAlly = readLineSync.keyInSelect(ally,)
+        console.log("")
+        console.log(ally[indexAlly] + " will be a reliable ally for you.")
+        
+    } else {
+        console.log("Ok if you're that confident in your abilities i cant go against that.")
+    }
+    
+}
+
+const bossInfo = () => {
+    if(readLineSync.keyInYN("Would you like to know why Chongaks is here? \n")){
+        console.log("")
+        console.log("Due to the destruction of the Well of Corruption, Kinshu known as the faceless one seeked vengeance!")
+        console.log("")
+        console.log("He promised to bring destruction to our world due to the amount of power he and his people have lost due to the well being destroyed.")
+        console.log("")
+        console.log("Kinshu made a deal with the being known as Seltharion, Seltharion has granted him power to unleash Chongaks onto our world.")
+        console.log("")
+        console.log("Kinshu had agree to sell the planet to Seltharion as soon as Chongaks is finished wiping every living thing that exist here.")
+        console.log("")
+        console.log("Together they used up all of their power to release Chongaks the corruptor. ")
+    } else {
+        console.log("That's fine lets finish setting up so we can end this nightmare.")
+    }
+}
+let gold = 100_000 
```

Comment:
This error is stopping the game from playing because an underscore cannot be used in a number